<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServoDetails</class>
 <widget class="QDialog" name="ServoDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>610</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="Arduino">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelArduino">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Arduino:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pin:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="Pin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cable Arduino-&gt;Terminal:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="CableArduinoTerminal">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Power Pin:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="PowerPin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>40</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Comment:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Comment:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Degrees:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Degrees:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Detach:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inverted:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Rest Degrees:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo Type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cable Terminal Slot:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cable Terminal-&gt;Servo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="CableTerminalServo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>570</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MinComment">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MaxComment">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="MinDegrees">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-120</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>-90</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="MaxDegrees">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-120</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="AutoDetach">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>500</number>
   </property>
   <property name="value">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="RestDegrees">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-120</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="Inverted">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="Enabled">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="ServoType">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>220</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="TerminalSlot">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="MinPos">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ZeroDegPos">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Pos:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Pos:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zero Degrees Pos:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="MaxPos">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>180</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Speed:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="MoveSpeed">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPin_19">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ms for 60 degree move.
 Control uses higher value of servoType and move speed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Arduino</tabstop>
  <tabstop>Pin</tabstop>
  <tabstop>PowerPin</tabstop>
  <tabstop>Enabled</tabstop>
  <tabstop>MinComment</tabstop>
  <tabstop>MaxComment</tabstop>
  <tabstop>MinPos</tabstop>
  <tabstop>MaxPos</tabstop>
  <tabstop>ZeroDegPos</tabstop>
  <tabstop>MinDegrees</tabstop>
  <tabstop>MaxDegrees</tabstop>
  <tabstop>AutoDetach</tabstop>
  <tabstop>Inverted</tabstop>
  <tabstop>RestDegrees</tabstop>
  <tabstop>ServoType</tabstop>
  <tabstop>MoveSpeed</tabstop>
  <tabstop>TerminalSlot</tabstop>
  <tabstop>CableArduinoTerminal</tabstop>
  <tabstop>CableTerminalServo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ServoDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServoDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
